docs/quickstart.md

```diff
@@ -84,8 +84,13 @@ JAVA_OPTS="-Dhttp-server.process-forwarded=true"
 
 # Start Trino servers
 for i in 1 2; do
-    docker run --name trino$i -d -p 808$i:8080 \
-        -e JAVA_TOOL_OPTIONS="$JAVA_OPTS" $TRINO_IMAGE
+    PORT=808$i
+    if ! lsof -i:$PORT > /dev/null; then
+        docker run --name trino$i -d -p $PORT:8080 \
+            -e JAVA_TOOL_OPTIONS="$JAVA_OPTS" $TRINO_IMAGE
+    else
+        echo "Port $PORT is already in use. Skipping trino$i."
```

Comment:
not sure if we wanna skip orchestrating trino here or just let it fail... I personally think letting it fail is better. Or we could programmatically find an open port...

Reply:
I thought it would be better off to re-run the script to add backends to gateway if they are running in background. Failing means that i need to stop/kill current trino containers everytime my db container is gone OR manually add them.

Reply:
Should we add a Warning in front of the message so it's clear that something is wrong with the port.